samples/unit-tests/indicator-current-price/recalculations/demo.js

```diff
@@ -101,13 +101,13 @@ QUnit.test('The currentPriceIndicator label should be visible, #14879.',
                 }
             },
             series: [{
-                compare: 'percent',
-                data: [100, 1, 1, 10, 1],
```

Comment:
This is very risky to change the config of the demo. In this particular case, perhaps `compare` and that dataset was necessary to reproduce the issue.

In general, I recommend using `chart.update()` after the last `assert()`, instead of alternating the initial config.
 
Just tested:
- https://jsfiddle.net/BlackLabel/b1e90d2x/1/ - with the old config: we can observe missing label - correct test
- https://jsfiddle.net/BlackLabel/b1e90d2x/2/ - with the new config: the label is visible - the purpose of the test is lost (the test is still testing something, but we leave the doors open for regressions)

Reply:
Sorry about that! I went too far with the unit tests simplification idea. I've restored the old test and added a series update like you suggested. Thank you for spotting that!